File: 6. Python/EX10.py
# Solicita a duração em segundos
seg = int(input("# Digite a duracao em segundos: "))

# Converte segundos em minutos
min = seg // 60
seg = seg % 60

# Converte minutos em horas
hor = min // 60
min = min % 60

# Imprime o resultado no formato HH:MM:SS
print(f"{hor:02d}:{min:02d}:{seg}")

File: 4. Java/EX10.java
package Apostila;

import java.util.Scanner;

public class EX10
{
    public static void main(String[] args) 
    {
        // Declara variáveis
        int seg, min, hor;
        
        // Cria Scanner
        Scanner ent = new Scanner(System.in);       
        
        // Entrada de dados
        System.out.print("# Digite a duracao em segundos: ");
        seg = ent.nextInt();
        
        // Calcula minutos e segundos restantes
        min = seg / 60;
        seg = seg % 60;
        
        // Calcula horas e minutos restantes
        hor = min / 60;
        min = min % 60;
        
        // Exibe o resultado no formato HH:MM:SS
        System.out.printf("%02d:%02d:%02d\n", hor, min, seg);
    }
}
